tests/nfs/test_earmark_feature.py

```diff
@@ -0,0 +1,156 @@
+from time import sleep
+
+from cli.ceph.ceph import Ceph
+from tests.cephfs.cephfs_utilsV1 import FsUtils
+from tests.nfs.nfs_operations import check_nfs_daemons_removed
+from tests.nfs.nfs_test_multiple_filesystem_exports import create_nfs_export
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def run(ceph_cluster, **kw):
+    """Verify readdir ops
+    Args:
+        **kw: Key/value pairs of configuration information to be used in the test.
+    """
+    fs_util = FsUtils(ceph_cluster)
+    config = kw.get("config")
+    nfs_nodes = ceph_cluster.get_nodes("nfs")
+    clients = ceph_cluster.get_nodes("client")
+    nfs_node = nfs_nodes[0]
+    operation = config.get("operation")
+    fs_name = config.get("cephfs_volume", "ceph_fs")
+    nfs_name = "cephfs-nfs"
+    nfs_export = "/export"
+    nfs_server_name = nfs_node.hostname
+    subvolume_group = "ganeshagroup"
+    subvolume_name = "subvolume"
+    earmark = config.get("earmark")
+
+    fs_util.create_subvolumegroup(
+        client=clients[0], vol_name=fs_name, group_name=subvolume_group
+    )
+    fs_util.create_subvolume(
```

Comment:
please use https://github.com/red-hat-storage/cephci/blob/main/cli/ceph/fs/sub_volume.py#L12 the cli approach